### ISSUE TITLE
Exclude some folders from validation

### ISSUE THREAD

**Issue by anatoly314** [angelozerr/eclipse-wtp-webresources#50]
For example following my question from SO <a href="http://stackoverflow.com/questions/32176088/undefined-css-javascript-image-file-in-eclipse">Undefined CSS/JavaScript/Image file in Eclipse</a> it will be great if I could exclude some resources from validation path like for example described here: <a href="http://stackoverflow.com/a/7931839/947111">How do I remove javascript validation from my eclipse project?</a>

**Comment by angelozerr** [comment]
@anatoly314 yes you are right. It looks like https://github.com/angelozerr/eclipse-wtp-webresources/issues/24

But I think the best mean is to provide complex Include/Exclude path like Eclipse JavaScript JSDT.

I'm thinking more and more that Eclipse should provide a plugin which provides generic include/eclude path.  https://github.com/angelozerr/tern.java uses https://github.com/angelozerr/minimatch.java to support include/exclude path.
